index.html

```diff
@@ -0,0 +1,2394 @@
+                      with comprehensive content marketing and advertising
+                      strategies.
+                    </p>
+                  </div>
+                </article>
+              </aside>
+  
+            </div>
+          </div>
+        </section>
+
+        <section class="in-the-magazine nw-bb bg-white container-fluid py-2 mb-3 mb-md-0">
+          <h1 class="section-heading-1 border-bottom-itm-1 nw-font-2 text-center px-md-3">
+            <span class="sect-icon nw-font-2 text-white nw-bg-red mr-2">N</span>
+            IN THE MAGAZINE
+          </h1>
+          <div class="row mx-md-0">
+            <article class="col-md-3 card border-0 rounded-0 py-2 mb-1">
+              <div class="card-image-container position-relative mb-2">
+                <img
+                  src="./images/may-22-2020-cover.jpg"
+                  class="card-img-top rounded-0"
+                  alt="Tedros and Xi Jimping"
+                />
+                <div
+                  class="position-absolute align-self-start story-tag nw-font-2 nw-text-red"
+                >
+                  MAY 22 ISSUE
+                </div>
+              </div>
+              <div class="card-body p-0">
+                <h2 class="card-title card-title-big text-center">
+                  <a href="#" class="story-link-small nw-font-2">
+                    SEE ALL FEATURES <i class="fa fas fa-chevron-right"></i>
+                  </a>
+                </h2>
+              </div>
+            </article>
+            <article class="col-md-3 card border-0 rounded-0 py-2 mb-3">
+              <div class="card-image-container position-relative mb-1">
+                <img
+                  src="./images/fe-privacy-banner.jpg"
+                  class="card-img-top in-the-magazine-img rounded-0 mb-2"
+                  alt="Tedros and Xi Jimping"
+                />
+                <div
+                  class="d-inline-block nw-bg-red story-tag-2 nw-font-2 text-white"
+                >
+                  COVER
+                </div>
+                <div class="d-inline-block story-tag-2 nw-font-2 nw-text-red">
+                  HEALTH
+                </div>
+              </div>
+              <div class="card-body p-0">
+                <h2 class="card-title card-title-small">
+                  <a href="#" class="story-link-small nw-font-2">
+                    Would You Let Government Track Your Smartphone To Help U.S.
+                    Reopen Sooner?
+                  </a>
+                </h2>
+              </div>
+            </article>
+            <article class="col-md-3 card border-0 rounded-0 py-2 mb-3">
+              <div class="card-image-container position-relative mb-1">
+                <img
+                  src="./images/fe-privacy-qa-crystal-watson-03.jpg"
+                  class="card-img-top in-the-magazine-img rounded-0 mb-2"
+                  alt="Tedros and Xi Jimping"
+                />
```

Comment:
- [ ] The size of this articles are not the same as the original page. Kindly fix that.

> Original
![image](https://user-images.githubusercontent.com/57414789/82128187-d2692100-977e-11ea-90f3-98a67d8d0649.png)
> Your page
![image](https://user-images.githubusercontent.com/57414789/82128191-ddbc4c80-977e-11ea-9185-d8b21e748813.png)